### PR TITLE
Developer-Guide.md editorial changes including:

### LINKED ISSUE #641: Review Developer-Guide.md for spelling, syntax, and other minor editorial changes.
**Which feature do you think can be improved?**

Correct the Developer-Guide.md grammar, spelling, and make other minor editorial changes.

**Before raising this feature request**

Have you looked at the [limitations document](https://github.com/kata-containers/documentation/blob/master/Limitations.md)?
Yes.

### PR DESCRIPTION
**Pull request by DougTW**
-Edited content for clarity
-Grammar-, syntax-, and spell-checking
-Corrected indentation
-Standardized lines to break at 78 characters
-Used 'Kata Containers' term consistently
Fixes #641
Signed-off-by: Doug Martin <doug.martin@intel.com>

### CODE REVIEW

File: Developer-Guide.md [hunk 1]
```diff
@@ -358,15 +395,22 @@ $ (cd /usr/share/kata-containers && sudo ln -sf "$image" kata-containers-initrd.
...
-You can build and install the guest kernel image as shown [here](https://github.com/kata-containers/packaging/tree/master/kernel#build-kata-containers-kernel).
+You can build and install the guest kernel image as shown
+[here](https://github.com/kata-containers/packaging/tree/master/kernel#build-kata-containers-kernel).
 
 # Install a hypervisor
 
-When setting up Kata using a [packaged installation method](https://github.com/kata-containers/documentation/tree/master/install#installing-on-a-linux-system), the `qemu-lite` hypervisor is installed automatically. For other installation methods, you will need to manually install a suitable hypervisor.
+When setting up Kata Containers using a [packaged installation method]
+(https://github.com/kata-containers/documentation/tree/master/install#installing-on-a-linux-system),
+the `qemu-lite` hypervisor is installed automatically. For other
```
Comment (egernst) [inline Developer-Guide.md:405]: while we're touching this, we should probably s/qemu-lite/qemu

**Comment by jodh-intel** [comment]
Hi @DougTW - Travis CI is failing. To resolve this, please can you change the first line of your commit to include the subsystem (say "docs:") and add a reference to the Fixes too (it needs to be in the commit, not just on the GitHub PR to keep Travis happy ;). Something like:

```
docs: Editorial changes to developer guide

Edited the developer guide content for clarity:

-Grammar-, syntax-, and spell-checking
-Corrected indentation
-Standardized lines to break at 78 characters
-Used 'Kata Containers' term consistently

Fixes #641

Signed-off-by: Doug Martin <doug.martin@intel.com>
```

**Comment by mvincerx** [comment]
@DougTW Can you please follow up with if you have questions on how to assure that this PR conforms to the CI? I can help you.  It would be good to finish this PR.

**Comment by jodh-intel** [comment]
Hi @DougTW - please could you rebase this PR as it is now conflicted?

**Pull request merged**